.github/workflows/ci.yml

```diff
@@ -0,0 +1,175 @@
+name: EFCore.PG CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+        dotnet: [ '3.0.100' ]
+        pg_major: [ 11, 10 ]
+        include:
+          - pg_major: 11
+            edb_version: '11.4-2'
+            postgis_version: '2.5.3'
+          - pg_major: 10
+            edb_version: '10.9-2'
+            postgis_version: '2.5.3'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Start PostgreSQL ${{ matrix.pg_major }} (docker)
+        if: startsWith(matrix.os, 'ubuntu')
+        env:
+          IMAGE: npgsql/postgres:${{ matrix.pg_major }}
```

Comment:
Just a thought... We go through all the trouble of manually configuring PostgreSQL below on Windows, (probably?) because we have no choice. Should we consider just reusing the same config stuff on Linux and obviate the need for our special docker image?

Don't get me wrong, I love the idea of doing this with docker - ideally we'd do it for Windows too. But given that we can't....

Just a thought.

Reply:
I've gone back and forth on this too. For now, I'd like to wait and see if as Actions stabilizes, we gain the ability to use service containers in the Windows VMs.

I'm also still entranced/perplexed by the Windows docker image. The PostgreSQL part worked great, it was just adding the PostGIS plugin that got complicated. I set it aside so that we could get Actions up and running sooner, but I expect to take another pass at it in November.

Reply:
No problem at all. I'm fine with anything that works.